srcpkgs/intel-compute-runtime/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'intel-compute-runtime'
+pkgname=intel-compute-runtime
+version=20.18.16699
+revision=1
+wrksrc=compute-runtime-${version}
+#create_wrksrc=yes
+archs="i686* x86_64*"
+build_style=cmake
+#configure_args=""
+#make_build_args=""
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends="pkg-config"
+makedepends="intel-gmmlib-devel intel-graphics-compiler libexecinfo-devel"
```

Comment:
`libexecinfo-devel` is only available on musl; make dependency conditional on musl if it is needed there.

Reply:
Commented out for now, will see later if it's good to add it for musl (after igt is fixed for musl)